db/migrate/20170421194857_create_github_contributors.rb

```diff
@@ -0,0 +1,18 @@
+class CreateGithubContributors < ActiveRecord::Migration
+  def change
+    create_table :github_contributors do |t|
```

Comment:
The `github_contributors` table has privilege to add additional columns to it. We can do the change you suggested but this will make the insert very slow since now we will be inserting in two tables. Do you think we should still make these changes?